tracr/io/dicom/read.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python
+"""
+A script that converts dicom files to an intensity voxel array
+The resulting array is 1000 x 1024 in cross section direction and 1014 "tall"
+"""
+
+import sys, os, re
+import numpy as np
+import dicom
+
+
+def _regularize_dcm_files(dcm_files):
+	# Are we reading a file, a list of files, or files from a directory?
+	# 1. Is dcm_files a string?
+	if isinstance(dcm_files, str):
+		# yes: either a single file or a directory
+		if os.path.isdir(dcm_files):
+			# create a list of DICOM files from the contents of the directory
+			# dcm_files = dcm_files.rstrip('/')
```

Comment:
If the object at this file path is a directory, stripping the trailing backslash(es) ensures POSIX compliant filenames; so was there a reason why you opted to comment this out?